mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterSubsystemXMLReader.java

```diff
@@ -324,23 +324,26 @@ private void parseSSL(XMLExtendedStreamReader reader, List<ModelNode> list, Path
         ParseUtils.requireNoContent(reader);
     }
 
-    private void parseSimpleLoadProvider(XMLExtendedStreamReader reader, ModelNode operation) throws XMLStreamException {
+    private void parseSimpleLoadProvider(XMLExtendedStreamReader reader, List<ModelNode> list, PathAddress parent) throws XMLStreamException {
+        PathAddress address = parent.append(SimpleLoadProviderResourceDefinition.PATH);
+        ModelNode operation = Util.createAddOperation(address);
+
         int count = reader.getAttributeCount();
         for (int i = 0; i < count; i++) {
             requireNoNamespaceAttribute(reader, i);
             XMLAttribute attribute = XMLAttribute.forName(reader.getAttributeLocalName(i));
             switch (attribute) {
                 case FACTOR: {
-                    // todo require?
-                    readAttribute(reader, i, operation, ProxyConfigurationResourceDefinition.Attribute.SIMPLE_LOAD_PROVIDER);
+                    readAttribute(reader, i, operation, SimpleLoadProviderResourceDefinition.Attribute.FACTOR);
                     break;
                 }
                 default: {
                     throw unexpectedAttribute(reader, i);
                 }
             }
         }
-        ParseUtils.requireNoContent(reader);
```

Comment:
Why remove this?  Doesn't this element still require no content?

Reply:
Fixed.